relnotes/expiry.feature.md

```diff
@@ -0,0 +1,6 @@
+* `ocean.time.timeout.model.ExpiryRegistrationBase`
+
+  New method, `drop`, provides same functionality as `unregister`, but also
+  removes the expiry from current list of fired events if it was present there.
+  This method makes possible to disable one expiry from timeout callback of
+  another expiry, even if they fire in the same time slot.
```

Comment:
This is a patch release though

Reply:
Yes, this is one of rare exception cases "have to add feature to fix critical bug" in my opinion. I would probably even make the method `package(ocean)` protected if I could, to stick with formalities. But I can't, so better at least document it.

Reply:
"even if they fire in the same time slot" -- here too. (Sorry to be pedantic, but I think this phrase adds unnecessary confusion.)